src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ReadObjectResponse.liquid

```diff
@@ -0,0 +1,33 @@
+﻿protected virtual async System.Threading.Tasks.Task<ObjectResponseResult<T>> ReadObjectResponseAsync<T>(System.Net.Http.HttpResponseMessage response, System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IEnumerable<string>> headers)
```

Comment:
We should rename this method to ReadJsonObjectResponseAsync as there plans to also support xml deserialization which would require a similar method